lib/iris/tests/unit/coords/__init__.py

```diff
@@ -79,7 +79,7 @@ def arrays_share_data(a1, a2):
 
 
 def lazyness_string(data):
-    # Represent the lazyness of an array as a string.
+    # Represent the laziness of an array as a string.
```

Comment:
I’m not sure about this one: the property of data being lazy is arguably a different concept from just, y’know, idleness.  So maybe we are OK to create our own word?

Reply:
Personally I think it's `laziness` or I guess we could:

1. invent a new word i.e., `lazyness`
2. keep the usage of `lazyness` as a word, but skip it for correction
3. opt for a compromise, such as `lazy-ness` (🤢)
4. change the language e.g., `Represent the lazy state of the array as a string.` i.e., always use `lazy` not `lazyness` nor `laziness`.

Reply:
I don’t have a strong opinion.  Though I note that `lazyness` is also used in the function name.  The function is public but undocumented…

Reply:
Seems like that horse has bolted... I'm just gonna back-out that change, and make `lazyness` a word to be ignored :+1:

Reply:
Although the code is peppered with usage of `laziness`... TBH I'm past the point of caring about this one

Reply:
🤷‍♀️

Reply:
See https://github.com/SciTools/iris/pull/5186/commits/25b8463ea62ee640333713a04d7511f2d1bf1b44